packages/salesforcedx-apex-debugger/package.json

```diff
@@ -56,7 +56,7 @@
     "vscode-uri": "1.0.1"
   },
   "scripts": {
-    "bundle:debugger": "esbuild ./src/adapter/apexDebug.ts  --bundle --outfile=dist/apexdebug.js --format=cjs --platform=node --minify",
+    "bundle:debugger": "esbuild ./src/adapter/apexDebug.ts  --bundle --outfile=dist/apexdebug.js --format=cjs --platform=node --external:vscode --minify",
```

Comment:
Do any of our other modules externalize vscode?

Reply:
A lot of them 🙂
![Screenshot 2024-03-29 at 11 24 01 AM](https://github.com/forcedotcom/salesforcedx-vscode/assets/139700604/792ffcaf-70ea-459e-bf19-97b2e80875fc)